docs/proposals/README.md

```diff
@@ -0,0 +1,64 @@
+# Agones Feature Proposals (AFPs)
```

Comment:
Does this README serves similar purpose of the files in that `0000-afp-process` folder? Seems like they both give an introduction of what AFP's process looks like.